bson/encode_test.go

```diff
@@ -854,6 +856,7 @@ func reflectionEncoderTest(t *testing.T) {
 				W: nil,
 				X: []map[struct{}]struct{}{},   // Should be empty BSON Array
 				Y: []map[struct{}]struct{}{{}}, // Should be BSON array with one element, an empty BSON SubDocument
+				Z: time.Now(),
```

Comment:
Small thing here, we should make a `now` variable that's equal to `time.Now()` to ensure that a slow running test doesn't fail here.